pom.xml

```diff
@@ -21,28 +21,48 @@
 
   <!-- Change: groupId, artifactId, version -->
   <groupId>org.superbiz</groupId>
-  <artifactId>tomee-rest-arquillian</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <artifactId>tomee-jaxrs-starter-project</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
 
   <packaging>war</packaging>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jboss.arquillian</groupId>
+        <artifactId>arquillian-bom</artifactId>
+        <version>${arquillian.version}</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <dependency>
-      <groupId>org.apache.openejb</groupId>
+      <groupId>org.apache.tomee</groupId>
       <artifactId>javaee-api</artifactId>
-      <version>${openejb.javaee.api}</version>
+      <version>${tomee.javaee.api}</version>
       <scope>provided</scope>
     </dependency>
+    <!-- NOTE geronimo-osgi-locator required by JAX-RS client from CXF unless JAX-RS client jar proceeds Java EE API jar -->
     <dependency>
-      <groupId>org.apache.openejb</groupId>
-      <artifactId>arquillian-tomee-embedded</artifactId>
-      <version>${tomee.version}</version>
+      <groupId>org.apache.geronimo.specs</groupId>
```

Comment:
This dependency, as well as cxf-rt-rs-client and johnzon-jaxrs, are required for running the test with TomEE remote. I've moved these dependencies to a comment block and documented when they need to be enable. You are correct that they are not required to run the test on embedded.